tuxemon/event/actions/npc_wander.py

```diff
@@ -21,25 +22,41 @@ class NpcWanderAction(EventAction):
     Script usage:
         .. code-block::
 
-            npc_wander <npc_slug> [frequency]
+            npc_wander <npc_slug>[,frequency][,c_v1][,c_v2]
 
     Script parameters:
         npc_slug: Either "player" or npc slug name (e.g. "npc_maple").
         frequency: Frequency of movements. 0 to stop wandering. If set to
             a different value it will be clipped to the range [0.5, 5].
             If not passed the default value is 1.
+        c_v1: coordinates upper_left vertex (eg 5,7)
+        c_v2: coordinates botton_right vertex (eg 7,9)
+
+        eg. npc_wander npc_slug,,5,7,7,9
 
     """
 
     name = "npc_wander"
     npc_slug: str
     frequency: Union[float, None] = None
+    x_v1: Union[int, None] = None
+    y_v1: Union[int, None] = None
+    x_v2: Union[int, None] = None
+    y_v2: Union[int, None] = None
```

Comment:
Consider using `Optional[int]` as a lighter typehint.
The typehint for `frequency` could be changed as well at the same time.